src/libcore/array.rs

```diff
@@ -42,8 +42,10 @@ use slice::{Iter, IterMut};
 /// instead.
 pub unsafe trait FixedSizeArray<T> {
     /// Converts the array to immutable slice
+    #[rustc_conversion_suggestion]
```

Comment:
`FixedSizeArray` is unstable and not in the prelude so it souldn't be suggested.